prov/gni/src/gnix_vc.c

```diff
@@ -1389,89 +1389,98 @@ int _gnix_vc_progress(struct gnix_vc *vc)
 	return FI_SUCCESS;
 }
 
+int _gnix_table_addr_to_map_addr(struct gnix_fid_av *int_av, fi_addr_t fi_addr, fi_addr_t *real_addr)
```

Comment:
I think this function can get better.  A few suggestions:
1. Move to gnix_av.c since it operates on an AV
2. Rename to show it operates on an AV _gnix_av_...
3. Generalize to handle all types of AVs (MAP and TABLE)
4. I think AV's map struct fi_addr_t to struct gnix_address, so the output parameter type should change